Rocket.Chat/Views/Cells/Chat/ChatMessageCell.swift

```diff
@@ -100,9 +100,9 @@ final class ChatMessageCell: UICollectionViewCell {
     func insertGesturesIfNeeded() {
         if self.longPressGesture == nil {
             let gesture = UILongPressGestureRecognizer(target: self, action: #selector(handleLongPressMessageCell(recognizer:)))
-            gesture.minimumPressDuration = 1
+            gesture.minimumPressDuration = 0.5
             gesture.delegate = self
-            contentView.addGestureRecognizer(gesture)
+            self.addGestureRecognizer(gesture)
```

Comment:
IMO, it would be more straightforward to add gesture to `self` since the whole view should respond to the gesture.